### ISSUE TITLE
Honor system

### ISSUE THREAD

**Issue by illyone** [LightsHope/server#1010]
Hello. I have a next qustion about honor system. Im ranking and 3 week in a row i was br1. Last week i was r13 with 90%, i take br1 and next week i get only 2% to my current progress. Can you please explain me, why i didn’t get r14 ? Thank you.

**Comment by stesve** [comment]
I assume you are playing on Anathema?

I took a quick glance at: http://realmplayers.com/PVPList.aspx?section=standings&realm=Ana&page=1

And it doesn't look like you had enough ppl in the pool to have a br1. 

Below 250 players in the pool and there is no br1.

**Comment by illyone** [comment]
Thats right. It means ranking is impossible nowadays? Or what i need to do for getting r14?

**Comment by berillis** [comment]
You need more players playing pvp.

**Comment by illyone** [comment]
Ok. My last question, is it still possible getting r14 if people below 250 ?

**Comment by bigfgeg** [comment]
No.
